gen2_demo.py

```diff
@@ -21,16 +21,31 @@
 parser.add_argument('-vid', '--video', type=str,
                     help="Path to video file to be used for inference (conflicts with -cam)")
 parser.add_argument('-lq', '--lowquality', action="store_true", help="Low quality visualization - uses resized frames")
+parser.add_argument('-d', '--depth', action="store_true", help="Use depth information")
 parser.add_argument('-cnnp', '--cnn_path', type=Path, help="Path to cnn model directory to be run")
 parser.add_argument("-cnn", "--cnn_model", default="mobilenet-ssd", type=str, choices=CNN_choices, help="Cnn model to run on DepthAI")
 parser.add_argument('-sh', '--shaves', default=13, type=int, help="Name of the nn to be run from default depthai repository")
 parser.add_argument('-cnn-size', '--cnn-input-size', default=None, help="Neural network input dimensions, in \"WxH\" format, e.g. \"544x320\"")
 parser.add_argument("-rgbf", "--rgb_fps", default=30.0, type=float, help="RGB cam fps: max 118.0 for H:1080, max 42.0 for H:2160. Default: %(default)s")
+
+def dct_range(arg):
```

Comment:
can you change it to the implementation like here - https://github.com/luxonis/depthai-experiments/blob/master/gen2-collecting-training-data/main.py#L11 ? 

This way, we will make a generic range checker for other uses too (and it's already implemented so it's copy-paste'able